customization-base/src/main/java/com/azure/autorest/customization/ClassCustomization.java

```diff
@@ -73,6 +73,20 @@ public String getClassName() {
         return className;
     }
 
+    /**
+     * Adds imports to the class.
+     *
+     * @param imports Imports to add.
+     * @return A new {@link ClassCustomization} updated with the new imports for chaining.
+     */
+    public ClassCustomization addImports(String... imports) {
```

Comment:
Another API we should add for class customization is to add a static block. `static { ... }`